README.md

```diff
@@ -1,11 +1,17 @@
 # What is multipass?
 
-It's a system that orchestrates the creation, management and maintenance
-of virtual machines and associated Ubuntu images to simplify development.
+Multipass is a lightweight VM manager for Linux and MacOS. It's designed for
+developers who want a fresh Ubuntu with a single command or hotkey. It uses
```

Comment:
I would say "a fresh Ubuntu virtual machine" here. Saying "a fresh Ubuntu with" seems like Ubuntu itself is the noun, but really should be the adjective in this case.

Also, we don't support a hotkey yet, so maybe we should leave that out for now?